views/email/email_partials/email_footer.hbs

```diff
@@ -0,0 +1,30 @@
+<br />
+<table align="center" width="100%" border="0" cellspacing="0" cellpadding="0" style="text-align: center; max-width: 603px; padding: 40px 10px 10px 10px; color: #5d5d5d; font-size: 10px;">
+  <tr>
+    <td style="border-top: 1px solid #e1e1e1;">
+      <p style="size: 11px; color: #5d5d5d; padding: 10px 0px 3px 0px; margin: 0px; font-weight: 300;">
+        We sent this email to {{ email }} because someone, hopefully you, opted into alerts from <a href="{{ serverUrl }}" style="color: #000000; font-weight: 500;">Firefox Monitor</a>.
+      </p>
+      <p style="padding: 0px 0px 3px 0px; margin: 0px;">
+        If you no longer want to receive alerts, please 
+        <a style="color: #000000; font-weight: 500;" href="{{ unsubscribeUrl }}" rel="noreferrer noopener">unsubscribe</a>.
+      </p>
+    </td>
+  </tr>
+  <tr>
+    <td valign="center" style="margin:0px; padding: 0px; height: 80px;">
+      <a href="https://www.mozilla.org" style="margin: 0px; padding: 0px;">
+        <img src="{{ serverUrl }}/img/favicon/favicon-196x196.c80e6abe0767.png" width="40" height="40" style="width: 40px; height: 40px;" />
```

Comment:
Hhmm... The hash in the favicon source seems a bit curious and prone to breakage.